src/components/Layout/Layout.css

```diff
@@ -26,25 +26,3 @@
   gap: 5px;
   font-weight: 500;
 }
-
-.LayoutMainContainer.layout__main {
-  margin-top: 64px;
-}
-
-@media only screen and (min-width: 1200px) {
-  .LayoutMainContainer.layout__main {
-    margin-top: 48px;
-  }
-}
-
-@media only screen and (min-width: 1200px) {
-  .ui.container {
-    width: 1124px;
-  }
-}
```

Comment:
This isn't needed on new CSS, right? asking because all other properties seems migrated but this one

Reply:
good catch